docs/admin-manual/user-priorities-negotiation.rst

```diff
@@ -362,6 +363,52 @@ the following ordered list of items.
               match list. The machine is removed from the list of
               resources to match (on this negotiation cycle).
 
+As described above, the *condor_negotiator* tries to match each job
+to all slots in the pool.  Assume that five slots match one request for
+three jobs, and that their ``NEGOTIATOR_PRE_JOB_RANK``, ``Job.Rank``, 
+and ``NEGOTIATOR_POST_JOB_RANK`` expressions evaluate (in the context 
+of both the slot ad and the job ad to the following values).
+
++------------+-------------------------+----------+-------------------------+
+|Slot Name   |  NEGOTIATOR_PRE_JOB_RANK|  Job.Rank| NEGOTIATOR_POST_JOB_RANK|
++============+=========================+==========+=========================+
+|slot1       |                      100|         1|                       10|
++------------+-------------------------+----------+-------------------------+
+|slot2       |                      100|         2|                       20|
++------------+-------------------------+----------+-------------------------+
+|slot3       |                      100|         2|                       30|
++------------+-------------------------+----------+-------------------------+
+|slot4       |                        0|         1|                       40|
++------------+-------------------------+----------+-------------------------+
+|slot5       |                      200|         1|                       50|
++------------+-------------------------+----------+-------------------------+
+
+Table 3.1: Example of slots before sorting
+
+Given that, the top three slots would be handed to the *condor_schedd* after
+sorting them first on ``NEGOTIATOR_PRE_JOB_RANK``, then sorting all ties based on ``Job.Rank``
+and any remaining ties sorted by ``NEGOTIATOR_POST_JOB_RANK``.  This
```

Comment:
I think the "Given that," here is a little deceptive, because nothing about the paragraph that follows depends on the details of the preceding table.  Perhaps something like

```suggestion
These slots would be sorted first on `NEGOTIATOR_PRE_JOB_RANK``, then sorting all ties based on ``Job.Rank``
and any remaining ties sorted by ``NEGOTIATOR_POST_JOB_RANK``.  After that, the first three slots would be
handed to the *condor_schedd*.  This
```